src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonNode.To.cs

```diff
@@ -25,7 +25,9 @@ public string ToJsonString(JsonSerializerOptions? options = null)
             Utf8JsonWriter writer = Utf8JsonWriterCache.RentWriterAndBuffer(writerOptions, defaultBufferSize, out PooledByteBufferWriter output);
             try
             {
+#pragma warning disable SYSLIB0060 // Type or member is obsolete
```

Comment:
I'm not comfortable with all the suppressions this change is forcing. Per my comment in https://github.com/dotnet/runtime/issues/108643 I don't believe the method should be obsoleted as such. Perhaps EB.Never is a better approach?

cc @stephentoub

Reply:
We should only obsolete tons of we don't want them used. If we're using them and believe the usage is correct and desirable, that's a clear sign we should not obsolete the method. Why was obsoletion recommended in the first place?